create.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kdam+Thmor+Pro&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <main>
+    <body>
+      <h1>CREATE</h1>
+      <h2 class="create">ADD Question</h2>
+      <section>
+        <article>
+          This is wehre you can create queistions!
+          <form action="Create" method="get">
+            <label for="question">Question:</label>
+            <input type="text" id="Question" name="question" /> <br />
+            <br />
+            <label for="question">Question:</label>
+            <input type="text" id="Question" name="question" />
+          </form>
+        </article>
+      </section>
+      <br />
+      <br />
+      <section>
+        <article>
+          This is wehre you type in Answer!
+          <form action="Create" method="get">
+            <label for="Answer">Answer:</label>
+            <input type="text" id="Answer" name="Answer" /> <br />
+            <br />
+            <label for="Answer">Answer:</label>
+            <input type="text" id="Answer" name="Answer" />
+          </form>
+        </article>
```

Comment:
I would suggest you to build one form which contains the input fields for question and answer, because answer and question are related. I think you don't need `article` and `section` here. You can apply your stylings with the form. And again don't use `br` elements for breaks between elements. Try to work with Flexbox, Grid or margin. You can nest `label` and `input` in a `p` element for example to group your input fields for easier styling. [See MDN for how to structure a form.](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form)